monitor/extensions/referencedata-wizards/src/main/java/org/datacleaner/monitor/server/wizard/dictionary/datastore/DatastoreDictionaryReferenceDataPage1.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.monitor.server.wizard.dictionary.datastore;
+
+import java.util.List;
+import java.util.Map;
+
+import org.datacleaner.monitor.server.wizard.shared.ReferenceDataHelper;
+import org.datacleaner.monitor.server.wizard.shared.datastore.DatastorePage1;
+import org.datacleaner.monitor.server.wizard.shared.datastore.DatastoreWizardSession;
+import org.datacleaner.monitor.shared.model.DCUserInputException;
+import org.datacleaner.monitor.wizard.WizardPageController;
+
+final class DatastoreDictionaryReferenceDataPage1 extends DatastorePage1 {
```

Comment:
We certainly can. What exactly do have on your mind? I can imagine: 
```
DatastoreDictionaryReferenceDataNamePage
DatastoreDictionaryReferenceDataSchemaPage
DatastoreDictionaryReferenceDataTablePage 
DatastoreDictionaryReferenceDataColumnPage
```
 but here I am missing the order. Perhaps I can leave the numbers: 
```
DatastoreDictionaryReferenceDataNamePage1
DatastoreDictionaryReferenceDataSchemaPage2
DatastoreDictionaryReferenceDataTablePage3
DatastoreDictionaryReferenceDataColumnPage4
```
but that makes me thinking that there might be something like DatastoreDictionaryReferenceDataNamePage2, DatastoreDictionaryReferenceDataNamePage3, ... as well which is not the case. 

So, what about: 
```
DatastoreDictionaryReferenceDataPage1Name
DatastoreDictionaryReferenceDataPage2Schema
DatastoreDictionaryReferenceDataPage3Table
DatastoreDictionaryReferenceDataPage4Column
```
?